.github/workflows/code-lintooor.yml

```diff
@@ -0,0 +1,26 @@
+name: ci
```

Comment:
seems like there's a bug in github actions where this worked for a hot sec, and is no longer running at all. i vote we merge this and hope that it eventually begins running